### ISSUE TITLE
More fonts or google fonts

### ISSUE THREAD

**Issue by uche40** [viliusle/miniPaint#107]
Hey Great app. But very limited text fonts. How do i add more fonts or can you integrate google fonts in the next update. Thanks

**Comment by viliusle** [comment]
Hey,
Yes, its limited, will add this to Todo list.

**Comment by uche40** [comment]
Ok cool, will be waiting on next update. And by the way, the text editing options section is way too hidden and hard to get to, on the lower right corner, consider moving it up .

**Comment by viliusle** [comment]
Closing, because its duplicate of #99.
